### ISSUE TITLE
Prompt template for LLM

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#26]
## User story
1. As a dev
2. I need to create a prompt template for the LLM,
3. So that when the LLM is given a chunk of text it can return extracted entities and relations from the text

## Acceptance criteria
* Devise a prompt to yield results from the LLM that are consistent (can start with industry partner's linked article and refine that prompt as needed)
* LLM should return valid .json that contains all entities and relations of the text chunk inputted

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
